apps/bare-expo/metro.config.js

```diff
@@ -0,0 +1,35 @@
+const path = require('path');
+const { createMetroConfiguration } = require('expo-yarn-workspaces');
+
+const blacklist = require('metro-config/src/defaults/blacklist');
+const config = createMetroConfiguration(__dirname);
```

Comment:
We use this config for all of the other apps -- is it that we load application code from `apps/ncl` that necessitates a custom config?